eng/pipelines/templates/steps/auth_dev_feed.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
```

Comment:
*nit*: This file name uses ```_``` as a separator, all the other YAML files use ```-```.

Reply:
> It looks like you are authing the dev feed in the build stage, is this to pull in dependencies that aren't in PyPI yet?

Absolutely.

Reply:
Done. Thanks for the catch Mitch.